content/php/concepts/math-functions/terms/decbin/decbin.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'decbin()'
+Description: 'Converts a decimal number to a binary number.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+  - 'Math Functions'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`decbin()`** function converts a decimal number to a binary number.
+
+The required parameter is the `number` that is used to specify the decimal value to convert. The result returns a `string` that contains the binary number of the decimal value.
```

Comment:
I would recommend moving this sentence down to the syntax section

Reply:
this is duly noted

thanks